chapters/introduction.tex

```diff
@@ -12,6 +12,7 @@ \section{Scope of the Specification}\label{scope-of-the-specification}
 The semantics of the Modelica language is specified by means of a set of
```

Comment:
Multiple-line sentence.

Reply:
Sentence-based line breaking of this file is taken care of in #3037:

Reply:
OK, resolving.